<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIP</class>
 <widget class="QMainWindow" name="PIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="inputBack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="output">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputPIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>enable</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="offsetX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="offsetY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scaleBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
